README.md

```diff
@@ -1,291 +1,201 @@
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
-
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Recipe-App
+\] ](#-Food_App_Rails
+)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🔭 Video Presentation](#video)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
+
+
+# 📖 [Budget_app_Rails]
+ <a name="about-project"></a>
+The Recipe App is a classic example of a foods website. It will be fully up and a functional website that will show the list of foods and empower users to interact with them by adding recipes,ingredients and generating shopping list.
+
 
 ## 🛠 Built With <a name="built-with"></a>
+Ruby
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
 <details>
-  <summary>Client</summary>
+  <summary>Server</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a https://www.ruby-lang.org/en/>Ruby</a></li>
   </ul>
 </details>
 
 <details>
-  <summary>Server</summary>
+  <summary>Database</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a>PostgreSQL</a></li>
   </ul>
 </details>
 
 <details>
-<summary>Database</summary>
+  <summary>Tests</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a>Rspec</a></li>
+    <li><a>Capybara</a></li>
   </ul>
 </details>
 
-<!-- Features -->
-
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+- **Authenticate user using device**
+- **Add new category**
+- **Add new spending**
+- **View new Transaction**
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+video presentation: https://www.loom.com/share/2db0feec45294fe9b11fbb03a853b642
```

Comment:
- [ ] Kindly show a piece of code while presenting on a section you are most proud of implementing. It is good to talk about a code section you are proud of and at the same time show the code to the viewers. Please redo the video and measure that part and all the required points to mention in your video. 👍🏽